### PR TITLE
ISPN-5097 Server jgroups configuration does not specify defaults for S3

### PR DESCRIPTION
**Pull request by alanfx**
https://issues.jboss.org/browse/ISPN-5097

Added new standalone configuration file (cloud.xml) with Cloud stacks

### CODE REVIEW

**Comment by alanfx** [comment]
@tristantarrant this PR replaces PR 4352

**Comment by slaskawi** [comment]
Looks good to me, but I would also like to get @tristantarrant +1.

File: server/integration/feature-pack/feature-pack-build.xml [hunk 1]
```diff
@@ -12,7 +12,11 @@
           template="configuration/standalone/template.xml" 
           subsystems="configuration/standalone/subsystems-clustered.xml" 
           output-file="standalone/configuration/clustered.xml" />
-        <domain 
+        <standalone
+          template="configuration/standalone/template.xml" 
+          subsystems="configuration/standalone/subsystems-cloud.xml" 
+          output-file="standalone/configuration/cloud.xml" />
+         <domain
```
Comment (danberindei) [inline server/integration/feature-pack/feature-pack-build.xml]: whitespace

**Comment by danberindei** [comment]
@alanfx Do you really need 2 commits? The 2nd commit seems to undo everything in the 1st one.

Also, we don't really like JIRA links in the commit message - the issue id is already on the first line, and the rest of the URL doesn't really add anything.

**Comment by alanfx** [comment]
@danberindei made your suggested changes and included the files needed to make this work!

**Comment by danberindei** [comment]
Integrated, thanks Alan!

**Pull request closed without merging**